core/statedb/statedb.go

```diff
@@ -612,3 +612,31 @@ func (s *StateDB) GetNextNftIndex() int64 {
 	}
 	return maxNftIndex + 1
 }
+
+func (s *StateDB) CacheGasAccountIndex(gasAccountIndex int64) {
+	_ = s.redisCache.Set(context.Background(), dbcache.GasAccountKey, gasAccountIndex)
+}
+
+func (s *StateDB) GetGasAccountIndex() (int64, error) {
+	gasAccountIndex := int64(-1)
+	_, err := s.redisCache.Get(context.Background(), dbcache.GasAccountKey, &gasAccountIndex)
```

Comment:
Will memory cache will be good enough?

Reply:
Yes. I think both are fine. The data is small and we also have db if the cache cannot be found.
Using memory cache will add another `cache`, kind of more complex.